AAI/AAIConnectProfile.md

```diff
@@ -365,27 +378,161 @@ Client --> ClearingHouse
+component "<b>Broker</b>\nservice" as Broker #FAFAD2
+component "<b>Client</b>\napplication" as Client #FAFAD2
+component "<b>Passport</b>\n<b>Clearinghouse</b>\nservice" as ClearingHouse #FAFAD2
+}
+
+VisaSource "n" --> "1" VisaRepository 
+VisaRepository "1" --> "0..1" VisaIssuer 
+VisaIssuer "0..1" --> "1" Broker 
+Broker "1" --> "1" Client 
+Client "1" --> "1" ClearingHouse
+
+@enduml
+
+> ... but I don't love it now that I've done it this way. (BTW, it's easy to do
+> in PlantUML; feel free to try to choose better labels than mine here.)
+>
+> We also talked about omitting the "unspecified" and "specified" boxes...
+
+@startuml
+skinparam componentStyle rectangle
+left to right direction
+
+component "<b>Visa Assertion Source</b>\norganization" as VisaSource
+database "<b>Visa Assertion</b>\n<b>Repository</b>\nabstract service" as VisaRepository
+component "<b>Visa Issuer</b>\nabstract service\n(optional)" as VisaIssuer
+component "<b>Broker</b>\nservice" as Broker
+component "<b>Client</b>\napplication" as Client
+component "<b>Passport</b>\n<b>Clearinghouse</b>\nservice" as ClearingHouse
+
+VisaSource "n" --> "1" VisaRepository 
+VisaRepository "1" --> "0..1" VisaIssuer 
+VisaIssuer "0..1" --> "1" Broker 
+Broker "1" --> "1" Client 
+Client "1" --> "1" ClearingHouse
+
+@enduml
+
+> ... but it still leaves a lot to explain.
+>
+> Here's a solution: Explode the one complex, overloaded diagram into multiple
+> simpler ones. Focus purely on the **roles** in the first diagram. Then explain
+> about flexibility of mechanisms. Then show two concrete non-normative
+> example diagrams. See below...
+>
+> And now we return you to your regularly scheduled spec draft...
+
+```
+</discussion>
+```
+
+### Flow of Assertions
+
+
+@startuml
+package "Flow of assertions" {
+
+skinparam componentStyle rectangle
+left to right direction
+
+component "<b>Broker</b>" as bro
+component "<b>Passport</b>\n<b>Clearinghouse</b>" as ch
+component "<b>Visa Assertion Source</b>" as vas
+component "<b>Visa Issuer</b>" as vi
+
+vas --> vi
+vi --> bro 
+bro --> ch
+
+}
+@enduml
+
+
+The above diagram shows how [Visa Assertions](#term-visa-assertion) flow from a [Visa Assertion Source](#term-visa-assertion-source)
+to a [Passport Clearinghouse](#term-passport-clearinghouse) that uses them. 
+
+Implementations may introduce clients, services, and protocols
+to provide the mechanisms to move the data between the
+[Visa Assertion Sources](#term-visa-assertion-source) and the [Broker](#term-broker).  These mechanisms are unspecified by the scope of this specification except that
 they MUST adhere to security and privacy best practices, such as those outlined
 in this specification, in their handling of protocols, claims, tokens and
-related data. The flow between these components (represented by black arrows)
+related data. The flow between these components
 MAY not be direct or conversely services shown as being separate MAY be
```

Comment:
```suggestion
MAY be indirect or conversely services shown as being separate MAY be
```

I think "MAY not be direct" seems like a suggestion that you should not use indirection here, which I don't think is the intent.